### PR TITLE
Add Alior Bank (PL) (538 branches)

### PR DESCRIPTION
**Pull request by starsep**

### CODE REVIEW

**Comment by iandees** [comment]
Can you add the bank category please?

**Comment by starsep** [comment]
> Can you add the bank category please?

@iandees Added in https://github.com/alltheplaces/alltheplaces/pull/6069/commits/4e7a556e6e76c0ea09e829cb48ed2a294f71eb16

File: locations/spiders/alior_bank_pl.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+class AliorBankPLSpider(Spider):
+    name = "alior_bank_pl"
+    item_attributes = {"brand": "Alior Bank", "brand_wikidata": "Q9148395"}
+    start_urls = ["https://www.aliorbank.pl/en/branches.html"]
+
+    def parse(self, response, **kwargs):
+        data = json.loads(response.xpath("//script[@id='branch']/text()").get())
+        for city, branches in data.items():
+            for branch in branches:
+                item = DictParser.parse(branch)
```
Comment (Cj-Malone) [inline locations/spiders/alior_bank_pl.py:19]: I don't think you're actually getting anything from DictParser here

```suggestion
                item = Feature()
```

Comment (starsep) [reply locations/spiders/alior_bank_pl.py]: Changed in be7739437d6c702695747e493754fbe4c188524d

**Pull request merged**